cEP-0032.md

```diff
@@ -0,0 +1,638 @@
+      NON-GCI & GCI participants will have a separate leaderboard.
+
+6. **coala contributor profile page**
+
+      Interactive contributor profile page with various features like the
+      statistics 2D or 3D graph that will show hike in contributions, activities,
+      overview of contributions and many more. Contributor will provided with
+      options - *Get promoted to developer*, *Apply as a mentor* etc.
+
+7. **Extend gamification leaderboard UI/UX and Information display**
+
+      Apart from displaying *number of commits*, *issues opened*, *reviews* and
+      *teams* on the leaderboard it can be made more creative and informative. To
+      make it more informative following information can be displayed -
+
+      * set of *issue labels*
+      * total contribution time
+      * *PRs Merged and Closed* count
+      * *Number of issues working on* count
+
+      And, to make it more creative we can add-
+
+      * *badges emojois* beside the username based on contributor data
+      * a search bar for searching a username
+      * advance filters for filtering out the leaderboard results
+
+8. **Automate assigning of issue to contributors**
+
+      Based on following checks, the issue will be assigned to avoid spam-
+  
+      * Not assigned to any other issue
+      * If assigned, the issue has a PR to close it
+
+      Since *corobo* does this work for us for assigning an issue to the
+      contributor, but it has been down from a long time and no idea when it
+      will up again for helping contributors. *corobo* provides features like
+      *invitation to newcomers*, *creating an issue*, *assigning an issue* and
+      many more. But due to maintaineance it is down. So, this feature on
+      community website can be treated for safe side. As well as it will help
+      contributors by not spamming on coala main gitter channel for getting
+      assigned on issue. Let's just keep that channel for coala related talk.
+
+### Improvements
+
+According to ​study published by adobe, 38% of people will stop engaging with a
+website if the content/layout is unattractive. Therefore, this section
+describes some major improvements in coala community website. Following are the
+imporvements:
+
+1. **UI/UX Design improvement**
+   1. Add *materialize.css*
+   2. Re-design all webpages
+   3. Gamification leaderboard
+        Changing format from a *div* block for each contributor to a table
+        format will give a better look from user point of view.
+
+2. **Update README.md providing more useful information**
+
+      1. Updated Header image
+
+            The new header image will symbolise the meaning of *creating,
+            supporting and growing* community.
+      2. *Why we need this webiste? (Role of community)*
+      3. Describing features of coala community website
+
+3. **Update coala newcomer docs about automation features**
+
+## coala [Webservices website](http://webservices.coala.io/)
```

Comment:
Have a section in this something like `Fixing the existing webservices website`, So that we get the real data at https://coala.io/#/getinvolved and community website can use the coala webservices instead of using [my raw git repo](https://github.com/coala/community/blob/07d6d512f84b6b6016ce9dd85f5255110b940f89/data/webservices.py#L12).

P.S. Fixing the existing webservices website with auto deployment on new prs merge should be of highest priority. Mention it in the beginning coding phase one timeline.

Reply:
> auto deployment on new prs merge

This should already work! I've seen merged prs and the pipeline is being triggered by repo maintainer. I was researching on this before bonding period, all i found is that some bugs were there which gitlab community fixed. It's still not confirmed whether its working or not. Also, there is no error in `.gitlab-ci.yaml` which isn't causing pipelines to get triggered automatically.
I would like to verify whether this issue still persists or not!

Reply:
> Fixing the existing webservices website

I will add this section

Reply:
>  Mention it in the beginning of coding phase one timeline.

Added in coding phase 1 milestone